scorep/__main__.py

```diff
@@ -22,7 +22,10 @@ def scorep_main(argv=None):
     no_default_threads = False
     no_default_compiler = False
     no_instrumenter = False
-    instrumenter_type = "profile"
+    if sys.version_info.major >= 3:
```

Comment:
The simple version check does not work for interpreters like pypy . By the way: the same is true for the c bindings in general. The profile interface works with pypy, but not the cProfile.

Reply:
I could make this more elaborate, e.g. try-import. However currently we have the same condition in setup.py, isn't that used for pypy too?